decidim-generators/spec/generators_spec.rb

```diff
@@ -68,6 +71,26 @@ module Decidim
           expect(File.read("#{test_app}/Gemfile"))
             .to match(/^gem "decidim-initiatives"/)
             .and match(/^gem "decidim-consultations"/)
+            .and match(/^gem "decidim-elections"/)
+            .and match(/^gem "decidim-conferences"/)
+            .and match(/^gem "decidim-templates"/)
+
+          # Checks that every table from a migration is included in the generated schema
+          schema = File.read("#{test_app}/db/schema.rb")
+          tables = []
+          dropped = []
+          Decidim::GemManager.plugins.each do |plugin|
+            Dir.glob("#{plugin}db/migrate/*.rb").each do |migration|
+              lines = File.readlines(migration)
+              tables.concat(lines.filter { |line| line.match? "create_table" }.map { |line| line.match(/(:)([a-z_0-9]+)/)[2] })
+              dropped.concat(lines.filter { |line| line.match? "drop_table" }.map { |line| line.match(/(:)([a-z_0-9]+)/)[2] })
+            end
+          end
+          tables.each do |table|
+            next if dropped.include? table
+
+            expect(schema).to match(/create_table "#{table}"|create_table :#{table}/)
```

Comment:
@mrcasals I've come out with this trick to test if the generated app has all the tables specified in the different modules. This works for the use case, if the `db` folder is not included in the gem, migrations are not copy and test fails.

What do you think?